pkg/core/native/notary.go

```diff
@@ -138,6 +139,9 @@ func (n *Notary) OnPersist(ic *interop.Context) error {
 			if tx.Sender() == n.Hash {
 				payer := tx.Signers[1]
 				balance := n.GetDepositFor(ic.DAO, payer.Account)
+				if balance == nil {
```

Comment:
How exactly does it happen?

Reply:
I was thinking about the case when main transaction has Notary as a sender, but I see that balance check during notary verification covers that. So close issue?